.github/workflows/combine-prs.yml

```diff
@@ -0,0 +1,163 @@
+#
+# Combine/group together PRs
+#
+# https://github.com/hrvey/combine-prs-workflow
+#
+
+name: 'Combine PRs'
+
+# Controls when the action will run - in this case triggered manually
+on:
+  workflow_dispatch:
+    inputs:
+      branchPrefix:
+        description: 'Branch prefix to find combinable PRs based on'
+        required: true
+        default: 'dependabot'
+      mustBeGreen:
+        description: 'Only combine PRs that are green (status is success)'
+        required: true
+        default: true
+      combineBranchName:
+        description: 'Name of the branch to combine PRs into'
+        required: true
+        default: 'combine-prs-branch'
+      ignoreLabel:
+        description: 'Exclude PRs with this label'
+        required: true
+        default: 'nocombine'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "combine-prs"
+  combine-prs:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
```

Comment:
Should we consider an ephemeral runner here instead? e.g. `automated-checks-ephemeral`

Reply:
Good point! Lets run this on `[self-hosted, automated-checks-ephemeral]` 👍